airsonic-main/src/main/java/org/airsonic/player/domain/TranscodeScheme.java

```diff
@@ -105,7 +105,7 @@ public static TranscodeScheme valueOf(int maxBitRate) {
 
     public static TranscodeScheme fromMaxBitRate(int maxBitRate) {
         for (TranscodeScheme transcodeScheme : TranscodeScheme.values()) {
-            if (maxBitRate == transcodeScheme.getMaxBitRate()) {
```

Comment:
Don't we want to get the closest number fo `maxBitRate`?

Reply:
Yes, and that's what it does, it gets the closes scheme that is less than or equal to `maxBitRate`